spec/duration.html

```diff
@@ -1004,7 +1011,7 @@ <h1>
       <emu-alg>
         1. Let _sign_ be ! DurationSign(_years_, _months_, _weeks_, _days_, _hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_).
         1. For each value _v_ of « _years_, _months_, _weeks_, _days_, _hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_ », do
-          1. If _v_ is not finite, return *false*.
+          1. If 𝔽(_v_) is not finite, return *false*.
```

Comment:
how could this ever happen, if this AO is constrained to only take integers? or is it if the integer is large enough that F turns it into Infinity?

Reply:
That's exactly it - you can have e.g. BalanceDuration(_years1_ + _years2_, ...)